src/components/ListItem/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { memo } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import Author from 'components/Author';
+import { Article } from 'types/Article';
+
+import styles from './styles.module.scss';
+
+interface Props {
+  article: Article;
+}
+
+function ListItem({ article }: Props) {
```

Comment:
```suggestion
function ListItem({ author, createdAt, favoritesCount, favorited, slug, title, description }: Props) {
```

Reply:
I would have to send each of the parameters in the component's call, I think it would be a bit expensive and if a new parameter appears later, I would have to modify all the files that call this component by adding the new parameter

Reply:
I agree with both, you could optimize it a little bit like renzo suggested, but sending all of those properties as props would be a little bit cumbersome and inconvenient, what you could do instead to not be using `article.propertyName` all over the place is to still just receive the article but destructure it in right after the initializing the component, like:
```
function ListItem({ article }: Props) {
const { author, createdAt, favoritesCount, favorited, slug, title, description } = article;
```

Reply:
Great!!